frontend/src/routes/nav.routes.tsx

```diff
@@ -25,9 +25,9 @@ export const DisplayedNavigationBarRoutes = [
     Icon: WebOutlined,
   },
   {
-    path: "select-difficulty",
-    element: <DifficultySelectionPage />,
-    label: "PeerPrep",
+    path: "select",
+    element: <SelectionPage />,
+    label: "Code Now!",
```

Comment:
Not sure if we should call this "Code Now!", or if "Collaborate" is a better word. Punctuations are usually omitted from navigation bars

Reply:
Will remove the punctuation for now